EXILED/Exiled.Events/Patches/Fixes/NWFixDetonationTimer.cs

```diff
@@ -0,0 +1,31 @@
+// -----------------------------------------------------------------------
+// <copyright file="NWFixDetonationTimer.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Fixes
+{
+    using System;
+    using System.Linq;
+
+    using GameCore;
+    using HarmonyLib;
+
+    /// <summary>
+    /// Fixes the issue where the game was not selecting the scenario with the nearest <see cref="AlphaWarheadController.DetonationScenario.TimeToDetonate"/> value.
```

Comment:
Could you  mention the original bug report(https://git.scpslgame.com/northwood-qa/scpsl-bug-reporting/-/issues/396) after this line?

Reply:
yes